projects/core/src/lib/components/home-page/home-page.component.html

```diff
@@ -0,0 +1,24 @@
+<div class="home">
+	<section *ngFor="let section of sections" [class]="section.name">
+		<h2 [class]="section.name+'__title'">{{section.title}}</h2>
+		<ul [class]="section.name+'__list'" *ngIf="section.benefits">
+			<li class="benefits__benefit" *ngFor="let benefit of section.benefits">{{benefit}}</li>
+		</ul>
+		<p [class]="section.name+'__content'" *ngIf="section.content">{{section.content}}</p>
+		<div class="how-it-works__instructions" *ngIf="section.instructions">
+			<p class="how-it-works__instruction" *ngFor="let instruction of section.instructions">
+				For {{instruction}} - please
+				<a class="link" href="" target="_blank">check here</a>
+			</p>
+			<p class="how-it-works__instruction"><a class="link" href="" target="_blank">FAQ</a></p>
+		</div>
+		<p class="feedback__content" *ngIf="section.feedback">
+			{{section.feedback}}
+			<a class="link" href="" target="_blank">here</a>
+		</p>
+		<hr
+			[class]="section.name+'__separator'"
+			[ngClass]="section===sections[sections.length-1] ? 'last-separator' : ''"
```

Comment:
may be better use NgForOF local variables "last"

![image](https://user-images.githubusercontent.com/69210975/220899793-00cfa0d1-71a9-4cb7-a1e4-9c6728132c55.png)

Reply:
done

Reply:
Done